backend/seafile/webapi.go

```diff
@@ -606,9 +606,14 @@ func (f *Fs) download(ctx context.Context, url string, size int64, options ...fs
 	// Build the http request
 	opts := rest.Opts{
 		Method:  "GET",
-		RootURL: url,
 		Options: options,
 	}
+	parsedURL, _ := url.Parse(downloadLink)
```

Comment:
This needs to check the error returned by `url.Parse` if err !=nil then `parsedURL` won't be valid.

Reply:
added error check